.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,41 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+assignees:
+  - octocat
```

Comment:
I am not sure if I understand this correctly, but i think this is not what we want? I think new reports should probably not be assigned at all by default.

Reply:
Definitely, I'll change it.